README.md

```diff
@@ -191,7 +193,7 @@ $ spark-shell \
 val compatibleSparkVersion = "2.4.5"
 assert(
     spark.version == compatibleSparkVersion,
-    s"The connector works just fine with Spark $compatibleSparkVersion")
+    s"The connector doesn't work with Spark $compatibleSparkVersion")
```

Comment:
Good catch, but...it should be `spark.version` instead (not `compatibleSparkVersion`).

Reply:
Changed in commit https://github.com/splicemachine/splice-machine-spark-connector/pull/31/commits/978249160be30c456b08c84acb8e6161476b9410